web-app/src/main/webapp/script.js

```diff
@@ -57,10 +57,33 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
 }
 
 window.onload = () => {
-  document.getElementById("form-container").style.display = "none";
   document.getElementById('report-button').addEventListener('click', showReportForm);
+  document.getElementById('back-icon').addEventListener('click', hideReportForm);
+  document.getElementById('menu-button').addEventListener('click', showMenu);
+  document.getElementById('close-menu').addEventListener('click', hideMenu);
 };
 
 function showReportForm() {
-  document.getElementById("form-container").style.display = "block";
+  document.getElementById('form-container').style.display = 'block';
+  let homeElements = document.getElementsByClassName('home');
+  for (let i = 0; i < homeElements.length; i++) {
```

Comment:
nit: for (element in homeElements) ?

Reply:
I don't think this works for nodeLists, I keep getting all the indices first before the element

Reply:
Should be possible: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of (see "Iterating over a DOM collection")